libobs/util/cf-lexer.c

```diff
@@ -71,22 +71,26 @@ char *cf_literal_to_str(const char *literal, size_t count)
 
 	if (count < 2)
 		return NULL;
-	if (literal[0] != literal[count-1])
+	if (literal[0] != literal[count - 1])
 		return NULL;
 	if (literal[0] != '\"' && literal[0] != '\'')
 		return NULL;
-
-	str = bmalloc(count - 1);
-	temp_src = literal;
+	//bzalloc so that the string is memset w/ 0s
+	str = bzalloc(count - 1);
+	//strip leading character
+	temp_src = literal + 1;
 	temp_dst = str;
-
-	while (*temp_src) {
+	//strip trailing character
```

Comment:
Oh, and additionally, please add a blank line before this.  Separating small code segments like this helps make the code more readable.  You erased the existing blank line here unnecessarily.  Keep things like that in mind for the future.